root/main/error/500.js

```diff
@@ -0,0 +1,97 @@
+/*
+ * @flow strict-local
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import EditLink from '../../static/scripts/common/components/EditLink';
+import bracketed from '../../static/scripts/common/utility/bracketed';
+import bugTrackerURL from '../../static/scripts/common/utility/bugTrackerURL';
+
+import ErrorLayout from './ErrorLayout';
+import ErrorEnvironment from './components/ErrorEnvironment';
+import ErrorInfo from './components/ErrorInfo';
+
+type Props = {
+  +$c: CatalystContextT,
+  +edits?: $ReadOnlyArray<EditT>,
+  +formattedErrors?: $ReadOnlyArray<string>,
+  +hostname?: string,
+  +useLanguages: boolean,
+};
+
+const Error500 = ({
```

Comment:
Shouldn’t filename match component’s name like we did so far? That is `Error500.js` instead of `500.js`.

Reply:
I just followed what we did for 404, but I'd prefer this too. I'll change it in an extra commit.